### ISSUE TITLE
Two threads concurrently tried to get the same name: master

### ISSUE THREAD

**Issue by drewbanin** [fishtown-analytics/dbt#1384]
### Issue description
This bug popped up on 0.13.0. Exact reproduction case TBD, though I suspect it has something to do with statements called from inside of models.

### Results
Error during parsing:
```
2019-04-01 11:33:53,380 (MainThread): Encountered an error:
2019-04-01 11:33:53,381 (MainThread): Runtime Error
  Two threads concurrently tried to get the same name: master
```

### System information
The output of `dbt --version`:
```
0.13.0
```

### Steps to reproduce
TBD

**Comment by Pratarshi** [comment]
### Updates:
1. the problem only happens while executing `dbt compile` with >1 threads
2. executing `dbt run` succeeds with >1 threads

**Comment by beckjake** [comment]
@Pratarshi do you have any examples for reproducing this, like sample models/macros that trigger it? Or nearby log messages?

**Comment by beckjake** [comment]
The fix for this was merged into the 0.13.1 branch in #1388
